reactor-core-micrometer/src/main/java/reactor/core/observability/micrometer/MicrometerMeterListenerConfiguration.java

```diff
@@ -93,9 +93,8 @@ static Tags resolveTags(Publisher<?> source, Tags tags) {
 		Scannable scannable = Scannable.from(source);
 
 		if (scannable.isScanAvailable()) {
-			List<Tag> discoveredTags = scannable.tagsDeduplicated()
-				.entrySet().stream()
-				.map(e -> Tag.of(e.getKey(), e.getValue()))
+			List<Tag> discoveredTags = scannable.tags()
+				.map(t -> Tag.of(t.getT1(), t.getT2()))
 				.collect(Collectors.toList());
 			return tags.and(discoveredTags);
 		}
```

Comment:
This code snippet occurred in three places; all three are updated in an identical manner. In each case the code is covered by a test that validates the desired deduplication logic ("last one wins"). (For brevity the new benchmark covers one of the two non-deprecated classes.)

Reply:
Check, I added a comment about the "last one wins" strategy. Let me know if you instead meant that the code snippets should reference each other and/or all point to the same benchmark class.